src/components/store/OrderSummary/index.tsx

```diff
@@ -26,22 +34,44 @@ const isOrderActionable = ({ status, pickupEvent }: PublicOrderWithItems): boole
   return true;
 };
 
-const OrderSummary = ({ order }: OrderSummaryProps) => {
+const OrderSummary = ({
+  order,
+  orderStatus,
+  futurePickupEvents,
+  pickupEvent,
+  reschedulePickupEvent,
+  cancelOrder,
+}: OrderSummaryProps) => {
   const items = getOrderItemQuantities(order.items);
 
   const totalCostWithoutDiscount = items.reduce((cost, item) => {
     return cost + item.quantity * item.option.price;
   }, 0);
 
-  const actionable = isOrderActionable(order);
+  const [modalOpen, setModalOpen] = useState<boolean>(false);
+  const [cancelModalOpen, setCancelModalOpen] = useState<boolean>(false);
+  const actionable = isOrderActionable({ ...order, status: orderStatus, pickupEvent });
```

Comment:
should change `isOrderActionable` to just take a `status` and `pickupEvent` since it only uses those fields, so you don't need to copy the rest of `order`